youtube/__init__.py

```diff
@@ -0,0 +1,192 @@
+import json
+import re
+import tempfile
+import time
+from concurrent.futures import ThreadPoolExecutor
+from pathlib import Path
+from typing import Any, Dict, List
+from urllib.parse import urlencode
+from urllib.request import Request, urlopen
+
+from albert import Action, Item, Query, QueryHandler, critical, info, openUrl  # pylint: disable=import-error
+
+
+md_iid = '0.5'
+md_version = '1.0'
+md_name = 'YouTube'
+md_description = 'Query and open YouTube videos and channels.'
+md_url = 'https://github.com/stevenxxiu/albert_youtube_user'
+md_maintainers = '@stevenxxiu'
+
+ICON_PATH = str(Path(__file__).parent / 'youtube.svg')
```

Comment:
this is the old icon. please use the new one. you can simply reuse the one from the websearch extension

Reply:
Sure, fixed.